sim-lib/src/lnd.rs

```diff
@@ -66,6 +67,10 @@ impl LndNode {
                     .map_err(|err| LightningError::GetInfoError(err.to_string()))?,
                 features: parse_node_features(features.keys().cloned().collect()),
                 alias,
+                networks: chains
```

Comment:
so CLN advertises just a single network while LND can advertise multiple? 🤦🏿

Reply:
Yes. And its why i made it a vec

Reply:
FUN LIGHTNING IS FUN OKAY.